BugByte-backend/src/main/java/com/example/BugByte_backend/repositories/userProfileRepository.java

```diff
@@ -128,6 +136,26 @@ public Integer getFollowingsCount(Long userId) {
         return count;
     }
 
+    public Integer getReputation(String userName) {
+        if (userName == null) {
+            throw new IllegalArgumentException("Username cannot be null");
+        }
+
+        try {
+            Integer rep = jdbcTemplate.queryForObject(
+                    SQL_GET_REPUTATION,
+                    new Object[]{ userName },
+                    Integer.class
+            );
+
+            System.out.println("Reputation is : " + rep);
+            return  rep != null ? rep : 0;
```

Comment:
- Remove unnecessary printing.
- What is the meaning of null rep? If the reputation is zero, it should be returned 0 from the database.

Reply:
valid point 
i will change it